module/size-speed-range/foundry/ranges.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import {displayMod} from '../../../lib/utilities.js';
+import {SSRT} from '../../../lib/gurps-foundry-ssrt-lib/src/js/SSRT.js';
+import {RangeBandTable} from '../domain/RangeBandTable.js';
+import convertToYards from '../../../lib/gurps-foundry-ssrt-lib/src/js/UnitsOfMeasure/Conversion.js';
+import parseDistance from '../../../lib/gurps-foundry-ssrt-lib/src/js/Parsing.js';
+import * as conversionMethods from '../../../lib/gurps-foundry-ssrt-lib/src/js/UnitsOfMeasure/Methods.js';
+
+const SYSTEM_NAME = 'gurps';
```

Comment:
This and a couple others were previously exported; don't export variables which will not be used in other files.